python/ray/autoscaler/_private/monitor.py

```diff
@@ -238,13 +239,32 @@ def get_latest_readonly_config():
             prom_metrics=self.prom_metrics,
         )
 
+    def get_cluster_resource_state(self):
```

Comment:
If a function is only called by other member functions within the same class, we typically prefix the function name with an underscore `_`.

```suggestion
    def _get_cluster_resource_state(self):
```